environments/staging/public.yml

```diff
@@ -49,9 +49,6 @@ localsettings:
   BANK_NAME: "RBS Citizens N.A."
   BANK_SWIFT_CODE: 'CTZIUS33'
   CELERY_PERIODIC_QUEUE: 'celery_null'
```

Comment:
not sure why this is `celery_null`

Reply:
Possibly to be sure we never run periodic queue on staging. I think there was some issue with not wanting to send emails because some prod domains were mirrored